src/patternfly/demos/Tabs/examples/Tabs.md

```diff
@@ -0,0 +1,814 @@
+      {{#> drawer drawer--id=(concat tabs--page-wrapper--id '-tabs') primary-detail-template--id=(concat tabs--page-wrapper--id '-tabs') drawer-panel--IsOpen="true" drawer--modifier="pf-m-inline-on-2xlsss pf-m-inline"}}
+        {{#> drawer-main}}
+          <!-- Content -->
+          {{#> drawer-content}}
+           {{> toolbar--template
+              toolbar--template--id=(concat tabs--page-wrapper--id '-toolbar')
+              toolbar--template--modifier="pf-m-page-insets"
+              toolbar--template--HasToggleGroup="true"
+              toolbar--template--HasFilter="true"
+              toolbar--template--HasSortButton="true"
+              toolbar--template--HasOverflowMenu="true"
+              toolbar--template--HasOverflowMenuSecondButton="true"
+              toolbar--template--OverflowButton1Text="Generate"
+              toolbar--template--OverflowButton2Text="Deploy"
+              toolbar--template--filterText="Name"}}
+            {{> divider}}
+            {{> tabs--table}}
+            {{#> pagination pagination--modifier="pf-m-bottom"}}
+              {{> pagination-options-menu id=(concat tabs--page-wrapper--id '-footer-pagination') options-menu--IsText="true" pagination-options-menu--modifier="pf-m-top"}}
+              {{> pagination-nav-content}}
+            {{/pagination}}
+          {{/drawer-content}}
+
+          <!-- Panel -->
+          {{#> drawer-panel drawer-panel--modifier="pf-m-width-33 pf-m-width-33-on-xl"}}
+            {{> tabs--panel-header tabs--panel-header--title="Node 2" tabs--panel-header--sub-title='<a href="#">siemur/test-space</a>'}}
+            {{#> drawer-body drawer-body--modifier="pf-m-no-padding"}}
+              {{> primary-detail-panel-tabs primary-detail-panel-tabs--modifier="pf-m-box pf-m-fill"}}
+            {{/drawer-body}}
+            {{#> drawer-body primary-detail-template--id=(concat tabs--page-wrapper--id '-tabs')}}
+              {{> primary-detail-panel-tab-content progress--modifier="pf-m-sm" primary-detail-panel-tab-content--HasLabels="true"}}
+            {{/drawer-body}}
+          {{/drawer-panel}}
+        {{/drawer-main}}
+      {{/drawer}}
+    {{/tab-content}}
+    {{#> tab-content tab-content--attribute=(concat 'aria-labelledby="' tab-content--id '-node-connectors-link" id="' tab-content--id '-node-connectors-panel"')}}
+      {{#> content}}
+        <p>Node connectors content</p>
+      {{/content}}
+    {{/tab-content}}
+  {{/page-main-section}}
+{{/tabs--page-wrapper}}
+```
+
+### Tables and tabs, auto width tabs
+```hbs isFullscreen
+{{#> tabs--page-wrapper tabs--page-wrapper--id="table-tabs-w-secondary-example"}}
+  {{#> page-main-section page-main-section--IsLimitWidth="true" page-main-section--modifier="pf-m-light"}}
+    {{#> title title--modifier="pf-m-2xl pf-u-mt-md"}}
+      Nodes
+    {{/title}}
+  {{/page-main-section}}
+  {{#> page-main-tabs page-main-tabs--IsLimitWidth="true" page-main-tabs--modifier="pf-m-light pf-m-tabs"}}
```

Comment:
You've got `.pf-m-light.pf-m-tabs` on a bunch of these - do you need those? Looks like you set the background color on `.pf-c-page__main-tabs` and that modifier isn't defined in the CSS anywhere.